etc/settings.yaml.in

```diff
@@ -43,6 +45,9 @@ execution:
     diffmeth:
       threads: 1
       memory: 30G
+    final_report:
+      threads: 1
+      memory: 20G
```

Comment:
This does not make sense. We should move every memory hungry calculation from the report to an external script.

Reply:
Yep, I also noticed that and think that in general it's better to move all memory demanding things to external scripts

Reply:
I found it strange as well, but the job was repeatedly crashing on the cluster, and I didn't have time to track down the reason, so this was the quickest and easiest way to get the reports I needed for the use case.